dsn.go

```diff
@@ -159,6 +159,7 @@ func (dsn Dsn) StoreAPIURL() *url.URL {
 
 // RequestHeaders returns all the necessary headers that have to be used in the transport.
 func (dsn Dsn) RequestHeaders() map[string]string {
+	// FIXME: why sentry_version=7?
```

Comment:
@kamilogorek is this something like an API version? When does it change?

I'd like to replace the magic number literal with a named constant 😄

Reply:
Correct, it's minimal version of the API that's compatible with the SDK. in JS we call it `SENTRY_API_VERSION`

Reply:
Thanks, I moved it into a constant.

Reply:
#113